model/clusters_mgmt/v1/cluster_migrations_resource.model

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of cluster migrations of a cluster.
+resource ClusterMigrations {
+    method List {
+        // Index of the returned page, where one corresponds to the first page. As this
+        // collection doesn't support paging the result will always be `1`.
+        in out Page Integer = 1
+
+        // Number of items that will be contained in the returned page. As this collection
+        // doesn't support paging or searching the result will always be the total number of
+        // migrations of the cluster.
+        in out Size Integer = 100
```

Comment:
Same comment as above @davidleerh @gdbranco

Reply:
I've made the changes to the comments here. See this MR for the backend changes: https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/merge_requests/9169